frontend/index_es.html

```diff
@@ -0,0 +1,551 @@
+            
+            // Title card
+            const PAGE_TITLE_TEXT = `${data.intro.translated_text.es.deployment_name} Pagina Inicial` // The sentence "Pagina Inicial" was hardcoded do lack of token translation in .json file. This will be discussed with Shankari.
+            $('#pageTitle').text(PAGE_TITLE_TEXT)
+            document.title = PAGE_TITLE_TEXT;
+            
+            // Navigation links
+            const PUBLIC_DASHBOARD_LINK = data.server.connectUrl.replace("api", "public");;
+            $('#publicDashboardLink').attr('href', PUBLIC_DASHBOARD_LINK);
+
+            // Deployment partner content text
+            const DEPLOYMENT_PARTNER_NAME = `${data.intro.deployment_partner_name}`;
+            $('#deploymentPartnerName').text(DEPLOYMENT_PARTNER_NAME);
+            
+            // Program or study specific differences
+            const PROGRAM_OR_STUDY = `${data.intro.program_or_study}`;
+            if(PROGRAM_OR_STUDY == 'study') {
+              $('#tokenType').text("an autogenerated ");
+            } else if(PROGRAM_OR_STUDY == 'program') {
+              $('#tokenType').text("a program-provided ");
+            }
+            $('#programOrStudy').text(PROGRAM_OR_STUDY);
+
+            // Study title content text
+            const STUDY_TITLE_TEXT = DEPLOYMENT_PARTNER_NAME + ` ${data.intro.translated_text.es.deployment_name}`;
+            $('#studyTitleOne').text(STUDY_TITLE_TEXT);
+            $('#studyTitleTwo').text(STUDY_TITLE_TEXT);
+
+            // Short textual description content text
+            const SHORT_TEXTUAL_DESCRIPTION_TEXT = `${data.intro.translated_text.es.short_textual_description}`;
+            $('#shortTextualDescription').text(SHORT_TEXTUAL_DESCRIPTION_TEXT);
+            
+            // Why we collect content text
+            const WHY_WE_COLLECT_TEXT = `${data.intro.translated_text.es.why_we_collect_es}`;
+            $('#whyWeCollect').text(WHY_WE_COLLECT_TEXT);
+            
+            // Join study text
+            // JOIN_CODE was already defined above
+            $('#joinStudyLink').text(JOIN_CODE)
+            $('#joinStudyLink').attr('href', JOIN_CODE)
+
+            // Token privacy content text
+            const TOKEN_PRIVACY_INFO_STUDY_TEXT = "You will log in using a token generated by the phone app. Only you know your token. Since we don’t collect email or phone number, there is no “Forgot password” functionality. NREL staff cannot identify your account or restore your data without your token. You should save this token in case you change phones.";
+            const TOKEN_PRIVACY_INFO_PROGRAM_TEXT = "You have been provided with a token by " + `${DEPLOYMENT_PARTNER_NAME}` + " to log in to the app. Please save this token in a safe place in case you need to reset your phone or change phones. If you have lost your token, please contact your admin at " + `${DEPLOYMENT_PARTNER_NAME}` + " for assistance.";
```

Comment:
These are still in English.

Reply:
We didn't use #tokenPrivacyInfo and basically hardcoded the entire text inside a paragraph. So we forgot to translate this part 😆. In our next PR we will utilize #tokenPrivacyInfo and have it translated.